inst/app_dir/server.R

```diff
@@ -112,7 +112,7 @@ shinyServer(function(input, output, session) {
   observe({
     is_numeric = is.numeric(input$kmeans_iters)
     is_integer = is.integer(input$kmeans_iters)
-    if (!is_numeric || !is_integer || input$clust_num < 0) {
```

Comment:
pri kopirovani validacie z ineho pola som nezmenil jednu premennu. raz som to opravil, a ocividne necommitol, prepisal